src/data/mappings/NewsStoryPageMapping.ts

```diff
@@ -0,0 +1,26 @@
+import { NewsStoryPageProps } from "@/types/index"
+import { NodeNewsStory } from "data/types/drupal"
+import { mediaImageDataService } from "@/components/media/dataService"
+import { generalEntityDataService } from "@/lib/delegators/generalEntityDataService"
+
+const NewsStoryPageMapping = function(node: NodeNewsStory): NewsStoryPageProps {
+      return {
+        type: node.type,
+        published: node.status,
+        title: node.title,
+        image: mediaImageDataService(node.field_media),
```

Comment:
These are still old-style. They would need to be replaced by a call directly to a Mapping function.